CONTRIBUTING.md

```diff
@@ -133,6 +133,42 @@ The project's goals and design philsophy are different from other projects, but
 * [Maki Icons Design Guidelines](https://www.mapbox.com/maki-icons/guidelines/)
 * [GNOME Icon Design Guildelines](https://developer.gnome.org/hig/stable/icons-and-artwork.html.en)
 
+## Typography
+
+This style uses the font "Noto" for a world-wide covering of scripts. The font
+size should be ≥ 10 (legibility).
+
+### Multi-line labels
+
+Additional to text-size we have to set text-wrap-width and text-line-spacing.
+For both, the absolute value is meaningless; it should be interpreted relative
+to the font size (em):
+```
+text-size: 10;
+text-wrap-width: 30; // 3.0 em
+text-line-spacing: -1.5; // -0.15 em
+```
+If text-size increases on higher zoom levels, the other parameters must also be
```

Comment:
I think the comma can be left out. "must also be" could also read "also have to be"